### PR TITLE
Send 'error' severity by default & document how to set it

### PR DESCRIPTION
**Pull request by kyrylo**

### CODE REVIEW

File: notice.go [hunk 1]
```diff
@@ -77,6 +78,13 @@ func NewNotice(e interface{}, req *http.Request, depth int) *Notice {
 		notice.Context[k] = v
 	}
 
+	switch n := e.(type) {
+	case *Notice:
+		if s := n.Context["severity"]; s != "" {
+			notice.Context["severity"] = s
+		}
+	}
```
Comment (vmihailenco) [inline notice.go]: I think this block should be removed unless there is a good reason.

File: README.md [hunk 2]
```diff
@@ -42,6 +42,19 @@ airbrake.AddFilter(func(notice *gobrake.Notice) *gobrake.Notice {
...
 
+## Setting severity
+
+[Severity](https://airbrake.io/docs/airbrake-faq/what-is-severity/) allows
+categorizing how severe an error is. By default, it's set to `error`. To
+redefine severity, simply overwrite `context/severity` of a notice object. For
+example:
+
+```go
+notice := airbrake.Notice("operation failed", nil, 3)
```
Comment (vmihailenco) [inline README.md]: NewNotice

File: notice.go (hunk 1 above)
Comment (kyrylo) [inline notice.go]: Without it I get the following:

```
• Failure [0.001 seconds]
Notifier
/Users/kyrylo/Code/gocode/src/github.com/airbrake/gobrake/notifier_test.go:151
  allows setting custom severity [It]
  /Users/kyrylo/Code/gocode/src/github.com/airbrake/gobrake/notifier_test.go:150

  Expected
      <string>: error
  to equal
      <string>: critical

  /Users/kyrylo/Code/gocode/src/github.com/airbrake/gobrake/notifier_test.go:149
------------------------------
```

Comment (vmihailenco) [inline notice.go]: You need to add something like this to NewNotice

```
notice, ok := e.(*Notice)
if ok {
    return notice
}
```

Even then it looks like a hack, but not a bad one.

Comment (kyrylo) [inline notice.go]: :+1: fixed

**Pull request merged**